src/ops/view.rs

```diff
@@ -28,12 +175,36 @@ pub(super) fn pretty_view(
     let warn = WARN;
     let note = NOTE;
 
+    let is_crate_from_crates_io = summary.source_id().is_crates_io();
+
     let mut shell = gctx.shell();
     let verbosity = shell.verbosity();
     let stdout = shell.out();
-    write!(stdout, "{header}{}{header:#}", package_id.name())?;
+
+    if is_crate_from_crates_io {
+        write!(
+            stdout,
+            "{header}{}{header:#}",
+            Hyperlink::Crate(package_id.name(), /* Version */ None)
+        )?;
+    } else {
+        write!(stdout, "{header}{}{header:#}", package_id.name())?;
+    }
```

Comment:
This feels redundant with the crates.io link

Reply:
Yes, I will remove the redundant one.

i.e. No need to make the `crate name` as a hyperlink 😉.

Reply:
Hi Ed~

I had sunset the hyperlink transformation about this one.
And sunset the transformation about the crate name(It will not be a link):
![image](https://github.com/hi-rustin/cargo-information/assets/24818903/7903be58-081b-4067-b076-24d144cd3b1d)